tools/lint-all

```diff
@@ -117,8 +117,12 @@ def check_pyflakes():
 
 def custom_check_file(fn, rules, skip_rules=[]):
     failed = False
-    for i, line in enumerate(open(fn)):
+    flag = False
+    f=open(fn,'r+')
```

Comment:
I guess we need the lastLine variable to store the lastline and check if it has the new line character or not. Any other way I can do it?